synapseclient/table.py

```diff
@@ -1028,6 +1028,32 @@ def __init__(self, values, rowId, etag=None, nameToColumnId=None):
             self.etag = etag
 
 
+def BuildTable(name, parent, values):
+    """
+    Build a Table object
+
+    :param name: the name for the Table Schema object
+    :param parent: the project in Synapse to which this table belongs
+    :param values: an object that holds the content of the tables
+      - a Pandas `DataFrame <http://pandas.pydata.org/pandas-docs/stable/api.html#dataframe>`_
+
+    :return: a Table object suitable for storing
+    """
```

Comment:
add code example in both the function documentation and the module documentation.